spec/betagouvbot/account_request_spec.rb

```diff
@@ -19,7 +19,7 @@
         it 'notifies the backing address' do
           is_expected.to include be_a_kind_of(BetaGouvBot::MailAction)
             .and(have_attributes(subject: 'Ton adresse @beta.gouv.fr'))
-            .and(have_attributes(recipients: [{ 'email' => 'bob@email.coop' }]))
+            .and(have_attributes(recipients: [{ email: 'bob@email.coop' }]))
```

Comment:
Are you quite sure that string vs symbol access makes no difference to the OVH API?